libqtile/scripts/migrate.py

```diff
@@ -144,6 +147,47 @@ def f(query, fro=fro, to=to):
     MIGRATIONS.append(f)
 
 
+# Number of migrations that have been removed.
+PREVIOUS_MIGRATIONS = 0
+
+MIGRATION_COUNT_PATTERN = re.compile(r"^migration_count\s*=\s*(\d+)\s*")
+
+
+def get_migration_count(config):
+    # this is fairly horrible, but there's no guarantee we can __import__() a
+    # config file in the current version, since it may not have been migrated
```

Comment:
Could you make it clear that this woud be due to moved imports within libqtile? I was about to comment saying "why not just make them fix it?" before I remembered the mention of this in the issue.

Reply:
Yes, I'll elaborate a bit here.